Omise.Tests/Resources/ChargeResourceTest.cs

```diff
@@ -120,7 +140,7 @@ public void TestPreAuthCreateChargeRequest()
         }
 
         [Test]
-        public void TestUpdateChargeRequest()
```

Comment:
What's changed here and in other lines?

Reply:
No change, its as mentioned before, the end of line problem